src/main/resources/portal.properties.EXAMPLE

```diff
@@ -100,6 +100,7 @@ microsoftlive.consumer.secret=
 
 ## SAML settings
 saml.sp.metadata.entityid=
+saml.sp.metadata.entityBaseURL=#{null}
```

Comment:
@holtgrewe Thanks for the PR! Why should this one be `#{null}`? Is that the same as empty string? Like why should this value have to be different from the other saml example values?

Reply:
ah i see never mind, you put it in the description, for keeping the old behavior. Makes sense

Reply:
maybe good to put a comment in the example file to clarify? Something like
```suggestion
# change this url if behind reverse proxy that handles SSL, see docs/Authenticating-Users-via-SAML.md
saml.sp.metadata.entityBaseURL=#{null}
```